pages/install/conda-install.rst

```diff
@@ -260,17 +263,18 @@ programs can be compiled as follows using the ``pkg-config`` build system:
 Windows Systems
 ---------------
 
-Installation folders for the C++ and Fortran 90 interface are:
+Installation folders for the C++ interface are:
 
 .. code:: shell
 
-   library files               path\to\conda\envs\ct-dev\Library\lib
-   C++ headers                 path\to\conda\envs\ct-dev\Library\include
-   samples                     path\to\conda\envs\ct-dev\share\cantera\samples
-   data files                  path\to\conda\envs\ct-dev\share\cantera\data
+   library files               path/to/conda/envs/ct-dev/Library/lib
+   C++ headers                 path/to/conda/envs/ct-dev/Library/include
+   samples                     path/to/conda/envs/ct-dev/share/cantera/samples
+   data files                  path/to/conda/envs/ct-dev/share/cantera/data
 
-The development package on Windows is experimental. While all libraries and headers are
-installed, compilation of custom programs is untested.
+The development package on Windows is experimental, with Fortran 90 currently not being
+supported. While libraries and headers are installed, compilation of custom programs is
+untested.
```

Comment:
```suggestion
The development package on Windows is experimental, with Fortran 90 currently not being
supported.
```
I'd suggest deleting that last sentence, it doesn't add anything to me and might scare away people who could test this for us :smile: